gammapy/irf/effective_area.py

```diff
@@ -64,6 +64,7 @@ class EffectiveAreaTable2D(IRF):
 
     tag = "aeff_2d"
     required_axes = ["energy_true", "offset"]
+    unit = u.Unit("m**2")
```

Comment:
Better use the defined units like this: `u.m**2`. No reason to go through string parsing.